docs/requirements.txt

```diff
@@ -3,4 +3,4 @@ sphinx-argparse==0.2.5
 sphinx-rtd-theme==0.4.3
 sphinxcontrib-websupport==1.1.0
 PyYAML==5.4.1
-
+jinja2<3.1.0
```

Comment:
This is the specific dep pinning defined in the attached issues remediation steps.